examples/hotplug_sensor_examples/displayio_layout_tablayout_hotplug_temp_sensor_datetime_fm_NTP_touch_and_gVars_test.py

```diff
@@ -0,0 +1,996 @@
+            print(
+                "Internal clock synchronized from NTP pool, now =",
+                myVars.read("default_dt"),
+            )
+
+
+if myVars.read("use_ntp"):
+    # Initialize the NTP object
+    ntp = NTP(esp)
+
+    location = secrets.get("timezone", location)
+    if myVars.read("my_debug"):
+        print("location (from secrets.h) = ", location)
+    if location == "Europe/Lisbon":
+        if myVars.read("my_debug"):
+            print("Using timezone Europe/Lisbon")
+        tz_offset = 3600
+    else:
+        tz_offset = 0
+
+    refresh_from_NTP()
+
+pixel = neopixel.NeoPixel(board.NEOPIXEL, 1, brightness=1)
+WHITE = 0xFFFFFF
+RED = 0xFF0000
+YELLOW = 0xFFFF00
+GREEN = 0x00FF00
+BLUE = 0x0000FF
+PURPLE = 0xFF00FF
+BLACK = 0x000000
+
+# ---------- Sound Effects ------------- #
+soundDemo = "/sounds/sound.wav"
+soundBeep = "/sounds/beep.wav"
+soundTab = "/sounds/tab.wav"
+
+# ------------ Touchscreen setup --------------- #
+# See: https://learn.adafruit.com/making-a-pyportal-user-interface-displayio/display
+display = board.DISPLAY  # create the display object
+display.rotation = 0
+# screen_width = 320
+# screen_height = 240
+screen_width = display.width
+screen_height = display.height
+# -------Rotate 0:
+# Note @PaulskPt dd 2022-05-13
+# After using a touchscreen calibration script, the values are as follows:
+# (XL, YU, XR, YD) are: (6935, 10496, 60127, 57631)
+ts = adafruit_touchscreen.Touchscreen(
+    board.TOUCH_XL,
+    board.TOUCH_XR,
+    board.TOUCH_YD,
+    board.TOUCH_YU,  # #calibration=((5200, 59000), (5800, 57000)),
+    calibration=((6815, 60095), (10520, 58007)),
+    size=(screen_width, screen_height),
+)  # was: screen_width, screen_height
+"""
+# If Rotate is 90:
+# -------Rotate 90:
+ts = adafruit_touchscreen.Touchscreen(board.TOUCH_YU, board.TOUCH_YD,
+                                      board.TOUCH_XL, board.TOUCH_XR,
+                                      calibration=((5200, 59000), (5800, 57000)),
+                                      size=(screen_height, screen_width))
+# If Rotate 180:
+ts = adafruit_touchscreen.Touchscreen(board.TOUCH_XR, board.TOUCH_XL,
+                                      board.TOUCH_YU, board.TOUCH_YD,
+                                      calibration=((5200, 59000), (5800, 57000)),
+                                      size=(screen_width, screen_height))
+
+# If Rotate 270:
+ts = adafruit_touchscreen.Touchscreen(board.TOUCH_XL, board.TOUCH_XR,
+                                      board.TOUCH_YD, board.TOUCH_YU,
+                                      calibration=((5200, 59000), (5800, 57000)),
+                                      size=(screen_height, screen_width))
+"""
+# -----------------------------------
+
+# create and show main_group
+main_group = displayio.Group()  # The Main Display Group
+
+display.show(main_group)
+
+# font = bitmap_font.load_font("fonts/Helvetica-Bold-16.bdf")
+font_arial = bitmap_font.load_font("/fonts/Arial-16.bdf")
```

Comment:
The Arial font BDF file that I found in another project used in a learn guide project did actually already have license info included in it. So that goes a long way toward changing my mind to leaning toward including the font file and it's license in the examples directory rather than making it optional.

Here is the one I found and tested with: https://github.com/adafruit/Adafruit_Learning_System_Guides/blob/main/PyPortal_OpenWeather/fonts/Arial-16.bdf